biochemist-python/chapters/File_Parsing.ipynb

```diff
@@ -0,0 +1,602 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "ls data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that there are two kinds of listings. There are three individual files (remdesivir.pdb, remdesivir.sdf and remdesivir.xyz) and there are two directories (PDB_files/ and enzymes/), which you can identify because their listings are followed by '/'.\n",
+    "\n",
+    "In order to parse a file, you must tell Python the location of the file, or the \"file path\". For example, you can see what folder your Jupyter notebook is in by typing `pwd` into a cell in your notebook and evaluating it. `pwd` stands for 'print working directory', and can also be used in your terminal to see what directory you're in.\n",
+    "\n",
+    "After evaluating the cell with `pwd`, you should see an output similar to the following if you are on Mac or Linux:\n",
+    "\n",
+    "/Users/YOUR_USER_NAME/Desktop/python-scripting-biochemistry/biochemist-python/chapters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "pwd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Or something like this if you are in Windows:\n",
+    "\n",
+    "C:\\Users\\YOUR_USER_NAME\\Desktop\\python-scripting-biochemistry\\biochemist-python\\chapters\n",
+    "\n",
+    "Notice that the file paths are different for these two systems. The Windows system uses a forward slash ('\\\\'), while Mac and Linux use a backslash ('/') for filepaths."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When we write a script, we want it to be usable on any operating system, thus we will use a python module called `os.path` that will allow us to define file paths in a general way.\n",
+    "\n",
+    "In order to get the path to the `4eyr.pdb` file in a general way, type"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "protein_file = os.path.join('data', 'PDB_files','4eyr.pdb')\n",
+    "print(protein_file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You should see something similar to the following:\n",
+    "\n",
+    "data/PDB_files/4eyr.pdb"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, we have specified that our filepath contains the 'data' and 'PDB_files' directory, and the `os.path` module has made this into a filepath that is usable by our system. If you are on Windows, you will instead see that a forward slash is used."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Absolute and relative paths\n",
```

Comment:
In the original lessons this was listed as a note, or aside. It simply shows up as a section in this version.

Reply:
fixed.